### ISSUE TITLE
[Placeholder] Selection rectangle

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/server#41788]
Add ability to select files and photos apps by dragging a selection rectangle

**Comment by jancborchardt** [comment]
Yup, would be a nice enhancement. We just need to see that it doesn’t interfere with drag & drop.
